versioned/tests/src/main/java/org/projectnessie/versioned/tests/AbstractCommits.java

```diff
@@ -535,10 +527,10 @@ public void commitConflictingOperations() throws Exception {
             () ->
                 commit("Conflicting Commit")
                     .fromReference(initialCommit)
-                    .put("t3", V_3_2.withId(t3), t3)
+                    .put("t3", V_3_2.withId(t3))
                     .toBranch(branch))
         .isInstanceOf(IllegalArgumentException.class)
-        .hasMessage("Key 't3' does not exist, but Put-operation has expectedValue");
+        .hasMessage("New value for new must not have a content iD");
```

Comment:
nit: the message looks odd to me :shrug:

Reply:
Maybe quote the key in the message? This particular message is just hard to read :)

Reply:
Also `iD` starts with a lower case ;)

Reply:
#6450